dolphinscheduler-ui/src/views/login/index.tsx

```diff
@@ -127,6 +143,19 @@ const login = defineComponent({
               {this.t('login.login')}
             </NButton>
           </div>
+          <div class={styles['form-model']} v-show={this.loginForm.ssoLoginUrl.length !== 0}>
+            <a href={this.loginForm.ssoLoginUrl} style="text-decoration:none">
+              <NButton
+                  class='btn-login'
```

Comment:
Seems you should change this class name, because it had exist the same class and e2e will find it  when login.
https://github.com/apache/dolphinscheduler/blob/8a152aebc70f57cc400c1cd8b16991817bed0d0e/dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/pages/LoginPage.java#L36-L50